Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -95,7 +96,7 @@ def extract_args
       switch "-f", "--force",
              description: "Overwrite the destination formula if it already exists."
 
-      named_args :formula, number: 2
+      named_args number: 2
```

Comment:
```suggestion
      named_args [:formula, :tap], number: 2
```

Even if we're not using the `named_args` to generate usage text, the types here define what shell completions are used. Keeping these here will allow users to tab-complete formula and tap names which is something we still want.

Reply:
I'm glad you see the same problems I do with the sentence structure!

Reply:
Do you mind if I push some changes to this branch? I think I have a fix for the `Parser` class that will improve the messaging and allow us to still have the formula and tap completions.

Reply:
Please feel free. ⭐ I've got to go and make dinner so I shan't be force-pushing over the top anymore!